frontend/public/components/utils/list-dropdown.jsx

```diff
@@ -175,22 +182,79 @@ ListDropdown.propTypes = {
   id: PropTypes.string,
 };
 
-const NsDropdown_ = (props) => {
-  const openshiftFlag = props.flags[FLAGS.OPENSHIFT];
-  if (flagPending(openshiftFlag)) {
-    return null;
+const useProjectOrNamespaceModel = () => {
+  const nsModel = modelFor('Namespace');
```

Comment:
Just import the static NamespaceModel directly. No need to retrieve it dynamically.

```suggestion
```